Add_user.php

```diff
@@ -1,39 +1,74 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
 		<title>Add User</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
-		<link rel ="stylesheet" href = "test10.css"> 
+		<!--Connected to css -->
+		<link rel ="stylesheet" href = "Add_user.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
-				<a href = "01_login_real.php"><img src = "images/logo8.png" height = "100"></a>
+				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
```

Comment:
I change link to be correct one. Because in all page the logo act as the home page. Si I want to make it consistence,